README.md

```diff
@@ -69,7 +64,7 @@ You can then connect to your local stack using the following URL : `http://fooba
 #### On iOS simulator
 
 - Create a cozy instance with the \*.cozy.tools:8080 format
-- Disable the local httpServer by setting to `true` disableGetIndex in `dev.js`
+- Disable the local httpServer by setting to `true` disableGetIndex in `src/constants/dev-config.ts`
```

Comment:
Should be done on both platforms

Reply:
Actually it was already done in Android so I just update it here.